internal/controller/volsync/vshandler.go

```diff
@@ -2206,3 +2204,14 @@ func (v *VSHandler) removeOCMAnnotationsAndUpdate(obj client.Object) error {
 
 	return v.client.Update(v.ctx, obj)
 }
+
+func (v *VSHandler) volumeModeForProtectedPVC(protectedPVC *ramendrv1alpha1.ProtectedPVC) *corev1.PersistentVolumeMode {
+	volumeMode := corev1.PersistentVolumeFilesystem
+	if util.IsRBDEnabledForVolSyncReplication(v.owner.GetAnnotations()) &&
+		protectedPVC.VolumeMode != nil &&
+		*protectedPVC.VolumeMode == corev1.PersistentVolumeBlock {
+		volumeMode = corev1.PersistentVolumeBlock
```

Comment:
Indentation here is confusing - maybe indent the conditions more to make the block more clear?

```go
if util.IsRBDEnabledForVolSyncReplication(v.owner.GetAnnotations()) &&
        protectedPVC.VolumeMode != nil &&
		*protectedPVC.VolumeMode == corev1.PersistentVolumeBlock {
	volumeMode = corev1.PersistentVolumeBlock
}
```

Reply:
Did you miss this comment?